style-about.css

```diff
@@ -0,0 +1,230 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;  
+}
+
+
+.header { 
+  font-family: 'Space Mono', sans-serif;
+  margin-bottom: 0;
+  font-weight: 700;
+  font-size: 48px;
+  letter-spacing: -2.14286px;
+  line-height: 48px;
+  color: #495567;
+}
+
+.text {
+  font-family: 'Lexend Deca', sans-serif;
+  width: 445px;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 25px;
+  margin-top: 24px;
+  margin-bottom: 40px;
+  color: #939CAA;
+}
+
+.button-cele3sectiuni {
+  font-family: 'Space Mono', sans-serif;
+  font-size: 15px;
+  font-weight: 700;
+  text-align: center;
+  color: #FFFFFF;
+  border: none;
```

Comment:
border should be: `border: 3px solid transparent;`

Reply:
fixed